ui/src/app/views/homepage/component.html

```diff
@@ -98,15 +98,23 @@
           </div>
         </div>
         <div class="row">
-          <div class="card add-card" (click)="addWorkspace()">
-            <div class="card-text">
-              Create a <br/> New Workspace <br/>
-              <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+          <div class="col-md-3" style="margin-bottom: 2rem;">
+            <div class="card add-card" (click)="addWorkspace()">
+              <div class="card-text">
+                Create a <br/> New Workspace <br/>
+                <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+              </div>
             </div>
           </div>
+          <div class="col-md-9">
+            <div class="row links" *ngIf="card.length>0" style="margin-left:4rem;margin-bottom:-2rem;">
+              Your recent work
+            </div>
+            <app-scroller [cards]=card></app-scroller>
```

Comment:
Nit: Can we make array variables plural? I think it would make the code easier to understand.